test/helper.rb

```diff
@@ -250,8 +250,10 @@ def md.directory?
       'pkginfo -l SUNWzfsr' => cmd.call('pkginfo-l-SUNWzfsr'),
       # solaris 11 package manager
       'pkg info system/file-system/zfs' => cmd.call('pkg-info-system-file-system-zfs'),
-      # dpkg-query package list
+      # dpkg-query all packages
       "dpkg-query -W -f='${db:Status-Abbrev} ${Package} ${Version}\\n'" => cmd.call('dpkg-query-W'),
+      # rpm query all packages
+      "rpm -qa --queryformat '%{NAME}   %{VERSION}-%{RELEASE}\\n'" => cmd.call('rpm-qa-queryformat'),
```

Comment:
I can see why he's doing it... but is there another character that is guaranteed to not be part of a RPM package name and version string that we can use as the delimiter instead? If not, I'm fine with the two spaces since that character combo should never be in a name or version.